src/plugins/embeddable/public/lib/panel/embeddable_panel.test.tsx

```diff
@@ -50,7 +50,7 @@ const embeddableReactFactory = new ContactCardEmbeddableReactFactory(
 const applicationMock = applicationServiceMock.createStartContract();
 const theme = themeServiceMock.createStartContract();
 
-actionRegistry.set(editModeAction.id, editModeAction);
+actionRegistry.set(editModeAction.id, editModeAction as unknown as Action<object>);
```

Comment:
I think Ideally we should be explicit that this is an editModeActionDefition and then use action registry to make it an `Action`

Reply:
Done. I've changed this line to create an action from the definition explicitly.

Yes, it should be like that, but unfortunately the way this test is written it is using some assumptions of internals of the UI Actions registry.